sentry-android-replay/src/main/java/io/sentry/android/replay/capture/BaseCaptureStrategy.kt

```diff
@@ -201,12 +202,12 @@ internal abstract class BaseCaptureStrategy(
 
         hub?.configureScope { scope ->
             scope.breadcrumbs.forEach { breadcrumb ->
-                if (breadcrumb.timestamp.after(segmentTimestamp) &&
-                    breadcrumb.timestamp.before(endTimestamp)
+                if (breadcrumb.timestamp.time >= segmentTimestamp.time &&
+                    breadcrumb.timestamp.time < endTimestamp.time
                 ) {
-                    // TODO: rework this later when aligned with iOS and frontend
                     var breadcrumbMessage: String? = null
-                    val breadcrumbCategory: String?
+                    var breadcrumbCategory: String? = null
+                    var breadcrumbLevel: SentryLevel? = null
                     val breadcrumbData = mutableMapOf<String, Any?>()
                     when {
```

Comment:
@romtsn what do you think about extracting this to a class so we can override it from RN and Flutter to have custom mapping?